tests/pytests/test_standard_suite.py

```diff
@@ -2502,13 +2140,38 @@ def test_olccd_energy_module(inp, dertype, basis, subjects, clsd_open_pmols, req
     ],
 )
 def test_olccd_energy_default(inp, dertype, basis, subjects, clsd_open_pmols, request):
-    method = "olccd"
+    runner_asserter(*_processor(inp, dertype, basis, subjects, clsd_open_pmols, request, "olccd", "energy"))
+
+
+def _processor(inp, dertype, basis, subjects, clsd_open_pmols, request, method, driver):
```

Comment:
Yes, it's the same cleanup I've applied to qcdb an qcng standard suite. Avoids repeating the same 15 lines for each test fn. It does make energy tests a little more complicated since errors need to be in a dict to behave like grad/hess, but that's a one-time cost.